### DISCUSSION TITLE
ARM supports

### DISCUSSION THREAD

**Discussion by JamesMenetrey** [icedland/iced#354] [Q&A] [answered]
Dear Iced developers,

I was wondering if the ARM assembler/disassembler is still a possibility as a future feature, or if [the idea](https://github.com/icedland/iced/issues/72) will not be realized?

With the advent of Apple which now uses its own ARM silicon, I can imagine a broader demand for such features. :)

Many thanks for your work and your time!
Cheers

**Comment by wtfsck** [comment, marked as answer]
Yes, it's on my to do list. Don't know when I'll have time though.

**Reply by JamesMenetrey** [reply]
What a great news @wtfsck, many thanks! Sorry to have posted an issue, I wasn't aware of the _Discussions_ feature.

While I have not planned on helping implementing the feature, feel free to ping when you need to perform some tests on real ARM hardware if needed.

Cheers

**Reply by alexrp** [reply]
Arm support is something I'd love to see as well.

I'd be happy to help fund the development of that support (and general Iced maintenance) if you ever decide to create a GitHub Sponsors profile or similar, by the way. 🙂

**Comment by teknalb** [comment]
:crossed_fingers:

**Comment by olukowski** [comment]
Hey! If needed, I can also try to help with the work required to support ARM, as I need it for one of my projects.

**Comment by xoofx** [comment]
Hey, I just finished a complete assembler/disassembler for ARM64 https://github.com/xoofx/AsmArm64
